apps/api/src/middleware/audit_logging.py

```
@@ -0,0 +1,388 @@
+
+    # User agent
+    user_agent = request.headers.get('User-Agent', 'unknown')
+
+    # Request ID (se disponível)
+    request_id = getattr(g, 'request_id', None) or request.headers.get('X-Request-ID', 'unknown')
+
+    # Usuário autenticado (se aplicável)
+    try:
+        verify_jwt_in_request(optional=True)
+        user_id = get_jwt_identity()
+    except:
```

Ruff (0.14.5)

Do not use bare `except`

(E722)